src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -813,23 +814,22 @@ internal static bool ConnectionExists(IEnumerable<KeyValuePair<string, string>>
 
         private void EnsureWorkerRuntimeIsSet()
         {
-            var workerRuntimeSettingValue = _secretsManager.GetSecrets().FirstOrDefault(s => s.Key.Equals(Constants.FunctionsWorkerRuntime, StringComparison.OrdinalIgnoreCase)).Value;
-            if (workerRuntimeSettingValue is not null)
+            if (_secretsManager.GetSecrets().Any(s => s.Key.Equals(Constants.FunctionsWorkerRuntime, StringComparison.OrdinalIgnoreCase))
```

Comment:
The `_secretsManager.GetSecrets().Any()` method could be more computationally intensive compared to checking the environment value (gut feeling, I don't have data to confirm this). Should we consider swapping the order of the checks?

Reply:
I think it depends on what we want to take precedent - does the local.settings file, or does the environment get priority?

Reply:
Good enough for me :)